
Showing logs for build [32mvicky-socs-test-python-jenkinsx-xhcjt-1[0m stage [32mfrom-build-pack[0m and container [32mstep-credential-initializer-zqdgp[0m
{"level":"warn","ts":1584529548.6224205,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1584529548.6236198,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mvicky-socs-test-python-jenkinsx-xhcjt-1[0m stage [32mfrom-build-pack[0m and container [32mstep-working-dir-initializer-8ddbv[0m
{"level":"warn","ts":1584529549.6349893,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1584529549.6390295,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mvicky-socs-test-python-jenkinsx-xhcjt-1[0m stage [32mfrom-build-pack[0m and container [32mstep-place-tools[0m

Showing logs for build [32mvicky-socs-test-python-jenkinsx-xhcjt-1[0m stage [32mfrom-build-pack[0m and container [32mstep-git-source-vicky-socs-test-python-jenkinsx-xhcjt-nt2d5[0m
{"level":"warn","ts":1584529576.75237,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1584529579.2688122,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/vicky-socs/test-python-jenkinsx.git @ v0.0.1 in path /workspace/source"}

Showing logs for build [32mvicky-socs-test-python-jenkinsx-xhcjt-1[0m stage [32mfrom-build-pack[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:de09fedbf9787a5e2fde0c06f206d8bf7a88233c
WARNING: no SHAs to merge, falling back to initial cloned commit

Showing logs for build [32mvicky-socs-test-python-jenkinsx-xhcjt-1[0m stage [32mfrom-build-pack[0m and container [32mstep-setup-jx-git-credentials[0m
Generated Git credentials file /workspace/xdg_config/git/credentials

Showing logs for build [32mvicky-socs-test-python-jenkinsx-xhcjt-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-python-unittest[0m

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK

Showing logs for build [32mvicky-socs-test-python-jenkinsx-xhcjt-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-container-build[0m
[36mINFO[0m[0000] Downloading base image python:onbuild        
2020/03/18 11:06:22 No matching credentials were found, falling back on anonymous
[36mINFO[0m[0002] Error while retrieving image from cache: getting file info: stat /workspace/sha256:43780a8431c53f4afd734ad795e03127c0c607337a60639c5903a51a03535c22: no such file or directory 
[36mINFO[0m[0002] Downloading base image python:onbuild        
2020/03/18 11:06:25 No matching credentials were found, falling back on anonymous
[36mINFO[0m[0004] Executing 3 build triggers                   
[36mINFO[0m[0004] Using files from context: [/workspace/source/requirements.txt] 
[36mINFO[0m[0004] Checking for cached layer 685249416972.dkr.ecr.ap-south-1.amazonaws.com/todo/cache:20ed5631d983ab35ddcc4224cde40d34ba4dcc68c4ee26635bbf30d01c3c2af6... 
[36mINFO[0m[0004] Using caching version of cmd: RUN pip install --no-cache-dir -r requirements.txt 
[36mINFO[0m[0004] Using files from context: [/workspace/source] 
[36mINFO[0m[0004] Using files from context: [/workspace/source/requirements.txt] 
[36mINFO[0m[0004] cmd: EXPOSE                                  
[36mINFO[0m[0004] Adding exposed port: 8080/tcp                
[36mINFO[0m[0004] Skipping unpacking as no commands require it. 
[36mINFO[0m[0004] Taking snapshot of full filesystem...        
[36mINFO[0m[0004] Using files from context: [/workspace/source/requirements.txt] 
[36mINFO[0m[0004] COPY requirements.txt /usr/src/app/          
[36mINFO[0m[0004] Taking snapshot of files...                  
[36mINFO[0m[0004] RUN pip install --no-cache-dir -r requirements.txt 
[36mINFO[0m[0004] Found cached layer, extracting to filesystem 
[36mINFO[0m[0004] Taking snapshot of files...                  
[36mINFO[0m[0005] Using files from context: [/workspace/source] 
[36mINFO[0m[0005] COPY . /usr/src/app                          
[36mINFO[0m[0005] Taking snapshot of files...                  
[36mINFO[0m[0005] Using files from context: [/workspace/source/requirements.txt] 
[36mINFO[0m[0005] COPY requirements.txt .                      
[36mINFO[0m[0005] Taking snapshot of files...                  
[36mINFO[0m[0005] ENV PORT 8080                                
[36mINFO[0m[0005] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0005] EXPOSE 8080                                  
[36mINFO[0m[0005] cmd: EXPOSE                                  
[36mINFO[0m[0005] Adding exposed port: 8080/tcp                
[36mINFO[0m[0005] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0005] ENTRYPOINT ["python"]                        
[36mINFO[0m[0005] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0005] CMD ["app.py"]                               
[36mINFO[0m[0005] No files changed in this command, skipping snapshotting. 
2020/03/18 11:06:28 pushed blob: sha256:76a575cfc4aa1c27ef0d73011a9919fa2b2f2e09e3ce99abe1c18147051bd459
2020/03/18 11:06:28 pushed blob: sha256:714912dad96aedac92cdb17ca15375a67db46094a9d25fd563c6fc61efddcbdd
2020/03/18 11:06:28 pushed blob: sha256:5e9ccc622f42f23940d24402ccca8ccab6b4112485454ad174ede9308d40c616
2020/03/18 11:06:28 pushed blob: sha256:e96bc319bce0351d1157092352bc5f85f8800c745388e60e86d06dec99285b45
2020/03/18 11:06:29 pushed blob: sha256:8b69db3637172aa85fb8471a6df2d7be41e038c649647986705cef4b8b9f2f6b
2020/03/18 11:06:29 pushed blob: sha256:e2c0fb34dff3a1d32346b92e53fb88a14d1987b402a2348cb379c039687fabe2
2020/03/18 11:06:29 pushed blob: sha256:490280dbcfb772eb5731f059e38fcfacaeb75b4d005af7c249fd51926db2bd62
2020/03/18 11:06:29 pushed blob: sha256:4d6de0f2b7101977f9944d727a27495ebc42b6c9660211701c18f7791efc3b76
2020/03/18 11:06:29 pushed blob: sha256:2d21c767035c4593de8bf7ad9781aa83ba110871d1a50871192f4e3480e6f11d
2020/03/18 11:06:30 pushed blob: sha256:e10b68fb77e6e91453cd4c1b02e2d77aa23a857cd8eb250617129d074478e1a5
2020/03/18 11:06:30 pushed blob: sha256:ece825d3308bbf98f708d95475ccd880497a1e2979ff4ef2534d49049a18488f
2020/03/18 11:06:31 pushed blob: sha256:06854774e2f35e4763103a446112c67322913a6177bea69eb49663e0166e6d6d
2020/03/18 11:06:32 pushed blob: sha256:1c7fe136a31e13a1c879245672184ced3b9fc590989ba42a522a57b071a9c1aa
2020/03/18 11:06:37 pushed blob: sha256:f0db43b9b8da254cebc77bf871006df01219e195b9853616b9f708f124a00f3a
2020/03/18 11:06:37 685249416972.dkr.ecr.ap-south-1.amazonaws.com/vicky-socs/test-python-jenkinsx:0.0.1: digest: sha256:ab320d6860853a07f2bf650af29e0bf03178a5c3a1655e537660d1efd4abbdec size: 2385

Showing logs for build [32mvicky-socs-test-python-jenkinsx-xhcjt-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-post-build[0m
no CVE provider running in the current ** namespace so skip adding image to be analysed

Showing logs for build [32mvicky-socs-test-python-jenkinsx-xhcjt-1[0m stage [32mfrom-build-pack[0m and container [32mstep-promote-changelog[0m
Converted /workspace/source/charts/test-python-jenkinsx to an unshallow repository
Generating change log from git ref 08b8dfdaf7615ea0e796cc8b1e035fd5037c97c0 => 1fcddcf52d4d1b14c482e37583499b00c26a2ffb
Associating user x-0 in users.jenkins.io with email jenkins-x@googlegroups.com to git GitProvider user with login -0 as emails match
Adding label jenkins.io/git-github-userid=-0 to user x-0 in users.jenkins.io
WARNING: Failed to enrich commit de09fedbf9787a5e2fde0c06f206d8bf7a88233c with issues: User.jenkins.io "x-0" is invalid: metadata.labels: Invalid value: "-0": a valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyValue',  or 'my_value',  or '12345', regex used for validation is '(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?')
Finding issues in commit messages using git format
WARNING: No release found for vicky-socs/test-python-jenkinsx and tag v0.0.1 so creating a new release
Updated the release information at https://github.com/vicky-socs/test-python-jenkinsx/releases/tag/v0.0.1
generated: /workspace/source/charts/test-python-jenkinsx/templates/release.yaml
Created Release test-python-jenkinsx-0-0-1 resource in namespace **
Updating PipelineActivity vicky-socs-test-python-jenkinsx-master-1 with version 0.0.1
Updated PipelineActivities vicky-socs-test-python-jenkinsx-master-1 with release notes URL: https://github.com/vicky-socs/test-python-jenkinsx/releases/tag/v0.0.1

Showing logs for build [32mvicky-socs-test-python-jenkinsx-xhcjt-1[0m stage [32mfrom-build-pack[0m and container [32mstep-promote-helm-release[0m
WARNING: No $CHART_REPOSITORY defined so using the default value of: http://jenkins-x-chartmuseum:8080
Adding missing Helm repo: storage.googleapis.com https://storage.googleapis.com/chartmuseum.jenkins-x.io
Successfully added Helm repository storage.googleapis.com.
Adding missing Helm repo: jenkins-x-chartmuseum http://jenkins-x-chartmuseum:8080
Successfully added Helm repository jenkins-x-chartmuseum.
WARNING: No $CHART_REPOSITORY defined so using the default value of: http://jenkins-x-chartmuseum:8080
Uploading chart file test-python-jenkinsx-0.0.1.tgz to http://jenkins-x-chartmuseum:8080/api/charts
Received 201 response: {"saved":true}

Showing logs for build [32mvicky-socs-test-python-jenkinsx-xhcjt-1[0m stage [32mfrom-build-pack[0m and container [32mstep-promote-jx-promote[0m
WARNING: prow based install so skip waiting for the merge of Pull Requests to go green as currently there is an issue with gettingstatuses from the PR, see https://github.com/jenkins-x/**/issues/2410
WARNING: No $CHART_REPOSITORY defined so using the default value of: http://jenkins-x-chartmuseum:8080
Promoting app test-python-jenkinsx version 0.0.1 to namespace **-staging
WARNING: 
WARNING: Waiting for the fork of ********/environment-non-prod-cluster-staging to appear...
Forked Git repository to https://github.com/********/environment-non-prod-cluster-staging
Created Pull Request: https://github.com/vicky-socs/environment-non-prod-cluster-staging/pull/1
Added label updatebot to Pull Request https://github.com/vicky-socs/environment-non-prod-cluster-staging/pull/1
got git provider status pending from PR https://github.com/vicky-socs/environment-non-prod-cluster-staging/pull/1
got git provider status pending from PR https://github.com/vicky-socs/environment-non-prod-cluster-staging/pull/1
got git provider status pending from PR https://github.com/vicky-socs/environment-non-prod-cluster-staging/pull/1
got git provider status pending from PR https://github.com/vicky-socs/environment-non-prod-cluster-staging/pull/1
got git provider status pending from PR https://github.com/vicky-socs/environment-non-prod-cluster-staging/pull/1
got git provider status pending from PR https://github.com/vicky-socs/environment-non-prod-cluster-staging/pull/1
got git provider status pending from PR https://github.com/vicky-socs/environment-non-prod-cluster-staging/pull/1
got git provider status pending from PR https://github.com/vicky-socs/environment-non-prod-cluster-staging/pull/1
got git provider status pending from PR https://github.com/vicky-socs/environment-non-prod-cluster-staging/pull/1
Pull Request https://github.com/vicky-socs/environment-non-prod-cluster-staging/pull/1 is merged at sha ef700b0f23edf6ef87d0ae61baf24d8cf895078e
Pull Request merged but we are not waiting for the update pipeline to complete!
WARNING: Could not find the service URL in namespace **-staging for names test-python-jenkinsx, **-test-python-jenkinsx, **-staging-test-python-jenkinsx
